modules/commons/src/main/resources/schemas/neurosciencegraph/commons/annotation/v0.2.0.json

```diff
@@ -0,0 +1,202 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.2.0",
+    {
+      "@base": "{{base}}/schemas/neurosciencegraph/commons/annotation/v0.2.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v1.0.0",
+    "{{base}}/schemas/neurosciencegraph/commons/selectors/v0.2.0"
+  ],
+  "@type": "nxv:Schema",
+  "shapes": [
+    {
+      "@id": "AnnotationShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "class": "oa:Annotation",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v1.0.0/shapes/EntityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:contribution",
+              "name": "Contribution",
+              "description": "Information about the agent involved in the creation of this Annotation.",
+              "node": "ContributionShape",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "oa:hasTarget",
+              "name": "Target",
+              "description": "The relationship between an Annotation and its Target.",
```

Comment:
Maybe provide description in sentence case (i.e. annotation and target lower case)